<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQTaskMethodWidget.ui,v $ </comment>
<comment>   $Revision: 1.2 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/03/07 19:38:00 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 - 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQTaskMethodWidget</class>
 <widget class="QWidget" name="CQTaskMethodWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>137</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="mpLblMethod" >
     <property name="text" >
      <string>Method</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="mpBoxMethod" />
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="mpLblMethodLog" >
     <property name="text" >
      <string>Log Verbosity</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QComboBox" name="mpBoxMethodLog" />
   </item>
   <item row="0" column="4" colspan="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>55</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="mpLblParameter" >
     <property name="text" >
      <string>Parameter</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="QTableWidget" name="mpTableParameter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column/>
    </widget>
   </item>
   <item row="1" column="5" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>45</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
